### PR TITLE
[#25] Add an Iceberg REST service demo

### PR DESCRIPTION
**Pull request by qqqttt123**
I added a demo for Iceberg REST service. Just modified some resources to make playground more stable.
Document pr is https://github.com/datastrato/gravitino/pull/2021/files

### CODE REVIEW

**Comment by qqqttt123** [comment]
cc @FANNG1

**Review by FANNG1 — APPROVED** [review]

**Comment by FANNG1** [comment]
LGTM, by the way, is there another PR showing how to use it?

**Comment by qqqttt123** [comment]
> LGTM, by the way, is there another PR showing how to use it?

You can see https://github.com/datastrato/gravitino/pull/2021/files

**Comment by jerryshao** [comment]
Do you need a doc to describe how to use it?

**Comment by qqqttt123** [comment]
> You can see https://github.com/datastrato/gravitino/pull/2021/files

You can see https://github.com/datastrato/gravitino/pull/2021/files

**Comment by qqqttt123** [comment]
> Do you need a doc to describe how to use it?

You can see https://github.com/datastrato/gravitino/pull/2021/files , I will sync the document from gravitino to gravitino-playground after this pr is merged.

File: init/gravitino/gravitino.conf [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2023 Datastrato Pvt Ltd.
```
Comment (jerryshao) [inline init/gravitino/gravitino.conf:2]: Please update the license header.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**